app/com/linkedin/drelephant/schedulers/AirflowScheduler.java

```diff
@@ -58,10 +62,14 @@ public AirflowScheduler(String appId, Properties properties, SchedulerConfigurat
   }
 
   private void loadInfo(String appId, Properties properties) {
-    // Update the 4 Ids
+    // examples:
+    // my_amazing_task_id
     _taskId = properties.getProperty(AIRFLOW_TASK_ID);
+    // 2016-06-27T01:30:00
```

Comment:
Good question. There is no limit. I think the solution is to implement something that truncates the name and appends a hash if it's too long.